### DISCUSSION TITLE
can't link string to clip

### DISCUSSION THREAD

**Discussion by bobpuffer1** [adieyal/comfyui-dynamicprompts#6] [Q&A] [answered]
just downloaded the latest version
Create a random prompt
contains wild card references to text files
string from random prompt won't link to clip For the clip text encode node

**Comment by bobpuffer1** [comment, marked as answer]
I was able to get it to link by converting the clip text to text input period now it seems to take my random prompt but with issues i'll bring that up in a different discussion

**Reply by adieyal** [reply]
Have a look at the example workflows. Converting to a text input is the right way to do it.
